packages/docs/app/layout.tsx

```
@@ -0,0 +1,54 @@
+        <Analytics />
+        <RootProvider
+          search={{
+            enabled: true,
+            SearchDialog: InkeepSearchBox,
+          }}
+          theme={{}}
+        >
+          {children}
+        </RootProvider>
+        <script
+          dangerouslySetInnerHTML={{
```

Biome (1.9.4)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)